plugins/woocommerce/client/blocks/tests/e2e/utils/admin/index.ts

```diff
@@ -6,24 +6,26 @@ import { Admin as CoreAdmin } from '@wordpress/e2e-test-utils-playwright';
 export class Admin extends CoreAdmin {
 	async visitWidgetEditor() {
 		await this.page.goto( '/wp-admin/widgets.php' );
-		await this.page.waitForFunction( () => {
-			window.wp.data
-				.dispatch( window.wp.preferences.store )
-				.set( 'core/edit-widgets', 'welcomeGuide', false );
-			return (
-				window.wp.data
-					.select( window.wp.preferences.store )
-					.get( 'core/edit-widgets', 'welcomeGuide' ) === false
-			);
-		} );
+		await this.page
```

Comment:
Hi @WunderBart I see this test is [being marked as flaky again](https://github.com/woocommerce/woocommerce/issues/48186#issuecomment-2743126698)

Is there a reason you changed back to closing the dialog "manually"? I know that doing it this way isn't 100% in the spirit of E2E tests, but given that the welcome guide is not the system under tests, and in light of the flakiness I think using the data store to guarantee the welcome guide is closed is acceptable.

Please let me know your thoughts 🙇🏼

Reply:
The previous implementation was extremely flaky when tested against v6.8-betaX as the Welcome Guide kept reopening, while the manual closing seemed to have fixed the issue in a backwards compatible way. We can try reverting this change, but we'll also need to ensure it's stable against 6.8 before merging. Does that make sense?

Reply:
> Welcome Guide kept reopening

This might be an issue to investigate on its own, but let's first try running against the beta3 and see if it's still flaky! 🤞

Reply:
Yes it does make sense, thanks for the explanation. How can I test against beta3 locally? I am happy to take care of this but would need help with setup, thanks

Reply:
Attaching the failure trace for reference: 
[trace.zip](https://github.com/user-attachments/files/19446813/trace.zip)

☝️ FWIW, it looks like the Welcome Guide never opened so the utility failed waiting on it. 

> How can I test against beta3 locally?

You need to update your wp-env config by replacing the `wordpress-latest` URL with https://wordpress.org/wordpress-6.8-beta3.zip. After that you'll need to run the following to update the environment:

```
pnpm --filter='@woocommerce/block-library' wp-env start --udpate
```

...or just reset the entire env with:

```
pnpm --filter='@woocommerce/block-library' env:restart
```

Reply:
BTW, when the PR is opened, you should also be able to manually run the `daily run` workflow against your branch which runs the E2E tests against the current, previous and next WP versions. The workflow can be triggered from here: 

https://github.com/woocommerce/woocommerce/actions/workflows/tests-daily-run.yml

![Screenshot 2025-03-25 at 12 23 50](https://github.com/user-attachments/assets/95141921-3624-4f67-b5bc-633f7b999ced)

Reply:
For now should we consider skipping or even just removing this test? The flakiness is a real blocker and this isn't a critical flow. What do you think @WunderBart ?

Reply:
Yeah, I think we can mark the tests as `fixme` as we did here https://github.com/woocommerce/woocommerce/pull/56517. Let's also create an issue to remember to investigate that test. How does that sound?